pyproject.toml

```diff
@@ -63,6 +63,7 @@ hypothesis = "^6.54.5"
 ipython = "^8.5.0"
 sphinx = ">=6,<9"
 babelgladeextractor = "^0.7"
+pytest-mock = "^3.14.0"
```

Comment:
Isn't it easier to use monkeypatch instead? I removed pytest-mock (which is a shallow wrapper around `unittest.mock`) from the dependencies a while ago.